app/Http/Controllers/Api/DatabasesController.php

```
@@ -2749,4 +2808,262 @@ public function action_restart(Request $request)
+            }
+        }
+
+        $targetTime = $request->input('target_time');
+
+        $restore = PgBackrestRestore::run($database, $execution, $targetTime);
+
+        return response()->json([
+            'message' => 'Database restore initiated.',
+            'restore_uuid' => $restore->uuid,
+        ], 200);
+    }
```

PHPMD (2.15.0)

The method restore_database is not named in camelCase. (undefined)

(CamelCaseMethodName)